contracts/test/vault/oethp-vault.plume.fork-test.js

```diff
@@ -0,0 +1,394 @@
+        .connect(rafael)
+        .transfer(oethpVault.address, oethUnits("10000"));
+
+      await oethpVault.rebase();
+      await _mint(strategist);
+
+      const vaultBalanceBefore = await weth.balanceOf(oethpVault.address);
+      const userBalanceBefore = await oethp.balanceOf(strategist.address);
+      const totalSupplyBefore = await oethp.totalSupply();
+
+      await oethpVault.connect(strategist).redeem(oethUnits("1"), "0");
+
+      const vaultBalanceAfter = await weth.balanceOf(oethpVault.address);
+      const userBalanceAfter = await oethp.balanceOf(strategist.address);
+      const totalSupplyAfter = await oethp.totalSupply();
+
+      expect(totalSupplyAfter).to.equal(totalSupplyBefore.sub(oethUnits("1")));
+      expect(userBalanceAfter).to.equal(userBalanceBefore.sub(oethUnits("1")));
+      expect(vaultBalanceAfter).to.equal(
+        vaultBalanceBefore.sub(oethUnits("1"))
+      );
+    });
+
+    it("Should allow only Governor to redeem", async () => {
+      const { governor, oethpVault, oethp, weth, rafael } = fixture;
+
+      // Add WETH liquidity to allow redeem
+      await weth
+        .connect(rafael)
+        .transfer(oethpVault.address, oethUnits("10000"));
+
+      await oethpVault.rebase();
+      await _mint(governor);
+
+      const vaultBalanceBefore = await weth.balanceOf(oethpVault.address);
+      const userBalanceBefore = await oethp.balanceOf(governor.address);
+      const totalSupplyBefore = await oethp.totalSupply();
+
+      await oethpVault.connect(governor).redeem(oethUnits("1"), "0");
+
+      const vaultBalanceAfter = await weth.balanceOf(oethpVault.address);
+      const userBalanceAfter = await oethp.balanceOf(governor.address);
+      const totalSupplyAfter = await oethp.totalSupply();
+
+      expect(totalSupplyAfter).to.equal(totalSupplyBefore.sub(oethUnits("1")));
+      expect(userBalanceAfter).to.equal(userBalanceBefore.sub(oethUnits("1")));
+      expect(vaultBalanceAfter).to.equal(
+        vaultBalanceBefore.sub(oethUnits("1"))
+      );
+    });
+
+    it("No one else can redeem", async () => {
+      const { rafael, nick, oethpVault } = fixture;
+
+      await oethpVault.rebase();
+
+      for (const signer of [rafael, nick]) {
+        await _mint(signer);
+        await expect(
+          oethpVault.connect(signer).redeem(oethUnits("1"), "0")
+        ).to.be.revertedWith("Caller is not the Strategist or Governor");
+      }
+    });
+  });
+
+  describe("Async withdrawals", function () {
+    it("Should allow 1:1 async withdrawals", async () => {
+      const { rafael, governor, oethpVault, weth } = fixture;
+
+      // Add WETH liquidity to allow withdrawal
+      await weth
+        .connect(rafael)
+        .transfer(oethpVault.address, oethUnits("10000"));
+
+      const delayPeriod = await oethpVault.withdrawalClaimDelay();
+
+      if (delayPeriod == 0) {
+        // Temporarily set to 10m if disabled
+        await oethpVault.connect(governor).setWithdrawalClaimDelay(
```

Comment:
any reason why not fork test with the on-chain set withdrawal delay? This way if the withdrawal delay changes the tests will also rightly fail